### DISCUSSION TITLE
请问x86 openwrt 下载那个包

### DISCUSSION THREAD

**Discussion by LEGION128** [AlistGo/alist#655] [Q&A] [answered]
是那个alist-linux-amd64.tar.gz吗？

**Comment by xhofe** [comment, marked as answer]
[alist-linux-musl-amd64.tar.gz](https://github.com/Xhofe/alist/releases/download/v2.1.0/alist-linux-musl-amd64.tar.gz)

**Reply by LEGION128** [reply]
> [alist-linux-musl-amd64.tar.gz](https://github.com/Xhofe/alist/releases/download/v2.1.0/alist-linux-musl-amd64.tar.gz)

你好我配置阿里云的时候他返回我这个 请问这是什么原因 我是openwrtx86设备
![image](https://user-images.githubusercontent.com/60326273/156589897-47c5c08b-4b6a-4f06-9a85-b542c4798924.png)

**Reply by xhofe** [reply]
dns的问题。

**Comment by fredxieyan** [comment]
请问是直接 .gz 文件怎么安装？之前都是上传的 .IPK 然后点击安装的。谢谢

![1](https://user-images.githubusercontent.com/77092602/227892854-8e563b02-53eb-48dc-a5f0-96c518e0ab25.png)

**Reply by LEGION128** [reply]
> 请问是直接 .gz 文件怎么安装？之前都是上传的 .IPK 然后点击安装的。谢谢
> 
> ![1](https://user-images.githubusercontent.com/77092602/227892854-8e563b02-53eb-48dc-a5f0-96c518e0ab25.png)

建议直接一键脚本

**Reply by haounihao** [reply]
怎么用一件脚本
